### DISCUSSION TITLE
What does prenative mean?

### DISCUSSION THREAD

**Discussion by unicomp21** [esfx/esfx#58] [Q&A] [answered]
I'm looking at the release tags, what does prenative mean?  @rbuckton

**Comment by rbuckton** [comment, marked as answer]
It was a prerelease tag for a build including native code to generate hash codes, but unfortunately is considered to be a higher semver value than the `-pre` versions so it gets picked up by dependencies. I plan to retire it once I ship 1.0.0, which will hopefully be very soon.
